src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -318,6 +319,18 @@ try
         rect.top = rect.bottom - 1;
         break;
     }
+    case CursorType::DoubleUnderscore:
+    {
+        // Use rect for lower line.
+        rect.top = rect.bottom - 1;
+
+        // Draw upper line directly.
+        D2D1_RECT_F upperLine = rect;
+        upperLine.top -= 2;
+        upperLine.bottom -= 2;

```

Comment:
Does this look correct at all DPIs or does it need to be adjusted by a scale factor?

Reply:
I switched "Custom scaling" to 200% in the settings. Is this the correct/preferred procedure?

Single underline is IMHO also thin:
![underscore](https://user-images.githubusercontent.com/4517789/98157496-ba768c80-1ed9-11eb-87c7-fec38bd01034.png)

And double underscore for comparison:
![doubleUnderscore](https://user-images.githubusercontent.com/4517789/98157559-cb270280-1ed9-11eb-80fc-fd0b9e49e2e4.png)

(Clicking the images helps.)

I'm open whether to scale or not to scale 😊

Reply:
Oof, can you file a follow on issue (or search the repo for an existing one) to make sure these things get scaled? I bet its just 1px and it looks bad for all options.

Reply:
Hm, I couldn't find one on searching. Maybe my boolean-fu is bad. @j4james this might be a thing you know about? I don't know why I think that... but something rings a bell in my head that you might have noticed cursors aren't DPI scaled...

Reply:
No. I was looking at the grid line rendering (underlines, overlines, etc.). Those do now track the font style and also scale appropriately, but I don't think the cursors do (at least not all of them).